src/main/java/com/openlattice/datastore/services/EdmService.java

```diff
@@ -124,6 +109,7 @@
     private final IMap<UUID, String>                                    names;
     private final IMap<UUID, AssociationType>                           associationTypes;
     private final IMap<EntitySetPropertyKey, EntitySetPropertyMetadata> entitySetPropertyMetadata;
+    private final IMap<EntityTypePropertyKey, EntityTypePropertyMetadata> entityTypePropertyMetadata;
```

Comment:
does this mapstore exist? /a postgres table where this data will live?

Reply:
That part I really don't understand.  It seems to work, but I don't know where in the database this data is stored... :-\

Reply:
You'll want to add a table in PostgresTable which specifies all the columns you need, and the data will be stored there. Then, you should add a mapstore which explains how to write data into the table (you can look at EntitySetPropertyMetadataMapstore for reference), and add it as a bean in MapstoresPod.